lib/buildScripts.coffee

```diff
@@ -0,0 +1,29 @@
+{CompositeDisposable} = require 'atom'
```

Comment:
I don't think this is required at all in this file. You should be able to just remove this. Test this though.